src/webtransport.js

```diff
@@ -802,10 +1125,42 @@ export class WebTransport {
   createUnidirectionalStream() {
     return this.sessionint.createUnidirectionalStream()
   }
+
+  getStats() {
+    return Promise.resolve({
+      timestamp: 0,
+      bytesSent: 0n,
+      packetsSent: 0n,
+      packetsLost: 0n,
+      numOutgoingStreamsCreated: 0,
+      numIncomingStreamsCreated: 0,
+      bytesReceived: 0n,
+      packetsReceived: 0n,
+      smoothedRtt: 0,
+      rttVariation: 0,
+      minRtt: 0,
+      datagrams: {
+        timestamp: 0,
+        expiredOutgoing: 0n,
+        droppedIncoming: 0n,
+        lostOutgoing: 0n
+      }
+    })
+  }
+
+  /** @returns {import('./dom').WebTransportReliabilityMode} */
+  get reliability () {
+    return 'reliable-only'
+  }
 }
 
 class Http3EventLoop {
-  static globalLoop = null
+  /** @type {Http3EventLoop | null} */
+  static globalLoop = Http3EventLoop.createGlobalEventLoop()
```

Comment:
Needs to be null. Since the object is resource-heavy and it is actually torn down, if no one uses it, and then it is again null.  But null guards may be unnecessary since they have to fail if it is null when it is called. It should not fail silently.

Reply:
Perfect, but the second chance in the commit will cause trouble.

Reply:
Do you mean this?:

```js
const loop = Http3EventLoop.createGlobalEventLoop()
```

`createGlobalEventLoop` only creates the loop if it doesn't exist so it's safe to call even if there's a loop already.

That said this is easier to read:

```js
const loop = Http3EventLoop.globalLoop ?? Http3EventLoop.createGlobalEventLoop()
```

..so I've changed it back to this.

As a future improvement maybe `createGlobalEventLoop` should throw if it's called twice?

Reply:
> As a future improvement maybe `createGlobalEventLoop` should throw if the loop already exists? My opinion on that is not that strong.

No, this would not be an improvement. Currently, the event loop works as follows.
If there is no event, one is created, but additionally, a guarding is started, which checks, if it is not needed anymore for some time (a couple of 10 seconds) and then it destructs itself. In this way, there is only an event loop if it is needed.